website/docs/docs/build/metricflow-time-spine.md

```diff
@@ -230,3 +258,52 @@ where date_day > dateadd(year, -4, current_timestamp())
 and date_hour < dateadd(day, 30, current_timestamp())
 ```
 </File>
+
+
+## Custom calendar
+
+<VersionBlock lastVersion="1.8">
+
+Being able to configure custom calendars, such as like a fiscal calendar, is available in [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) or dbt Core [v1.9 and above](/docs/dbt-versions/core). 
+
+To access this feature, [upgrade to Versionless](/docs/dbt-versions/versionless-cloud) or dbt Core v1.9 and above.
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+If you use a custom calendar in your organization, such as a fiscal calendar, you can configure it in MetricFlow using its date and time operations. 
+
+This is useful for calculating metrics based on a custom calendar, such as fiscal quarters or weeks. Use the `custom_granularities` key to define a non-standard time period for querying data, such as a `retail_month` or `fiscal_week`, instead of standard options like `day`, `month`, or `year`. This feature provides more control over how time-based metrics are calculated.
+
+
+### Why use a custom calendar model?
+Custom date transformations can be complex, and organizations often have unique needs that can’t be easily generalized. Creating a custom calendar model allows you to define these transformations in SQL, offering more flexibility than native transformations in MetricFlow. This approach lets you map custom columns back to MetricFlow granularities, ensuring consistency while giving you control over the transformations.
+
+### Calendar model requirements
+
+To use a custom calendar model, ensure it meets the following requirements:
+
+- The model must be at a daily grain and joinable to other date columns using `date_day`.
+- Columns must align with expected data types for a given granularity (for example, `quarter` should be a date).
+- Support period-over-period calculations.
+- Support cumulative metric windows using custom granularities (for example, cumulative fiscal year-to-date bookings).
+
+### Add custom granularities
+
+To add custom granularities, the Semantic Layer supports custom calendar configurations that allow users to query data using non-standard time periods like `fiscal_year` or `retail_month`. You can define these custom granularities by modifying your model's YAML configuration like this:
+
+<File name="models/_models.yml">
+
+```yaml
+models:
+ - name: my_time_spine
+   description: "my favorite time spine"
+   time_spine:
+      standard_granularity_column: date_day
+      custom_granularities:
+        - name: fiscal_year
```

Comment:
presume this is case sensitive, right?

Reply:
Spec looks good but linking back to the source of truth just to be safe: https://github.com/dbt-labs/dbt-semantic-interfaces/issues/338#issuecomment-2313655240